src/components/media-player/ha-media-player-browse.ts

```diff
@@ -173,81 +309,83 @@ export class HaMediaPlayerBrowse extends LitElement {
-                                    : "none",
-                                })}
-                              >
-                                ${this._narrow && currentItem?.can_play
-                                  ? html`
-                                      <ha-fab
-                                        mini
-                                        .item=${currentItem}
-                                        @click=${this._actionClicked}
-                                      >
-                                        <ha-svg-icon
-                                          slot="icon"
-                                          .label=${this.hass.localize(
-                                            `ui.components.media-browser.${this.action}-media`
-                                          )}
-                                          .path=${this.action === "play"
-                                            ? mdiPlay
-                                            : mdiPlus}
-                                        ></ha-svg-icon>
-                                        ${this.hass.localize(
-                                          `ui.components.media-browser.${this.action}`
-                                        )}
-                                      </ha-fab>
-                                    `
-                                  : ""}
-                              </div>
-                            `
-                          : html``}
-                        <div class="header-info">
-                          <div class="breadcrumb">
-                            <h1 class="title">${currentItem.title}</h1>
-                            ${subtitle
-                              ? html` <h2 class="subtitle">${subtitle}</h2> `
-                              : ""}
-                          </div>
-                          ${currentItem.can_play &&
-                          (!currentItem.thumbnail || !this._narrow)
+                  ? html`
+                      <div
+                        class="header ${classMap({
+                          "no-img": !currentItem.thumbnail,
+                          "no-dialog": !this.dialog,
+                        })}"
+                        @transitionend=${this._setHeaderHeight}
+                      >
+                        <div class="header-content">
+                          ${currentItem.thumbnail
                             ? html`
-                                <mwc-button
-                                  raised
-                                  .item=${currentItem}
-                                  @click=${this._actionClicked}
+                                <div
+                                  class="img"
+                                  style=${styleMap({
+                                    backgroundImage: currentItem.thumbnail
+                                      ? `url(${currentItem.thumbnail})`
```

Comment:
This doesn't work. It needs to be signed. That's what the intersection observer was doing.

Reply:
It was also rewriting the brand pictures to support dark mode.

Reply:
fixing in next push